muxpi/parse_image.sh

```diff
@@ -72,23 +72,32 @@ set +e
 7z x $IMAGESPATH/*.zip -o$IMAGESPATH 
 set -e
 
-mv $IMAGESPATH/$VARIANT* $IMAGESPATH/$VARIANT
-
+ls $IMAGESPATH
+mv $IMAGESPATH/$VARIANT*rootfs* $IMAGESPATH/$VARIANT
 echo "Compresing \"$VARIANT\"..."
 (cd $IMAGESPATH; tar -czf $VARIANT.tar.gz $VARIANT)
 if [ $? == 0 ]; then
    echo "Image has finished compressing"
 fi
 
-echo "{\"${VARIANT}\":\"\"}" > $MUXPIPATH/map.json
+echo "{\"${VARIANT}\":\"\"}" > ./map.json
 echo "Json map is ready"
 
 echo "Flashing muxpi for \"$ARCHVARIANT\""
 if [ "$ARCHVARIANT" == intel ]; then
    scp -i ~/.ssh/build_slave_key $IMAGESPATH/$VARIANT.tar.gz muxpi@172.31.173.165:~/artifacts
-   scp -i ~/.ssh/build_slave_key $MUXPIPATH/map.json muxpi@172.31.173.165:~/artifacts
-   scp -i ~/.ssh/build_slave_key $MUXPIPATH/vagrant-cookbook/muxpi/nuc/flash.sh muxpi@172.31.173.165:~/scripts
-   scp -i ~/.ssh/build_slave_key $MUXPIPATH/vagrant-cookbook/muxpi/nuc/ethup.sh muxpi@172.31.173.165:~/scripts
-   scp -i ~/.ssh/build_slave_key $MUXPIPATH/vagrant-cookbook/muxpi/nuc/validation-NUC.sh muxpi@172.31.173.165:~/scripts
-   ssh -i ~/.ssh/build_slave_key muxpi@172.31.173.165 "~/scripts/flash.sh $DUT_IP $VARIANT.tar.gz $NEPTUNE_IMAGE"
+   scp -i ~/.ssh/build_slave_key ./map.json muxpi@172.31.173.165:~/artifacts
```

Comment:
Is the IP fixed to 172.31.173.165? Would it be possible to have the IP address defined in a separate file, which could be sourced by the script? For example, have
File: my-settings.conf
MUXPI_IPADDR=172.31.173.165

Then in parse_image.sh, source the configuration file...
source my-settings.conf
...
scp -i ~/.ssh/build_slave_key ./map.json muxpi@$MUXPI_IPADDR:~/artifacts

Reply:
The IP address is set as static and is not expected to change.

Reply:
Yes, IPs of muxpis are static, however IP addresses could be as a parameter in script and then defined in Jenkins.

Reply:
Great to have it configurable via variable.